model_analyzer/record/metrics_manager.py

```diff
@@ -430,19 +430,15 @@ def _load_model_variants(self, run_config: RunConfig) -> bool:
         """
         Loads all model variants in the client
         """
-        # TODO TMA-1487: Make BLS and ensemble both load all composing model variants first
         for mrc in run_config.model_run_configs():
+            # Load all composing model variants first, and then the parent model
+            for composing_config_variant in mrc.composing_config_variants():
```

Comment:
Why are we no longer checking if mrc is a bls model?

Reply:
We need to always load composing models now, including for ensembles, since we go through the API and don't want default configs to be loaded.

Reply:
Conclusion: all models need to load sub models first so the bls check is pointless now.